src/pages/Help/ZHelp.css

```diff
@@ -0,0 +1,40 @@
+.flex_container {
```

Comment:
You should enlarge the padding of the ZHelp component, to make the numbers of the list fit inside. Because when you are in a mobile view they get out of the rectangle. 
![image](https://user-images.githubusercontent.com/25295489/61708940-5b295300-ad4e-11e9-819a-bcd052bdab09.png)

Reply:
cool, all there!!, try to layout it bit better